composer.json

```diff
@@ -47,11 +47,13 @@
     },
     "require-dev": {
         "drupol/php-conventions": "^3.0",
+        "sebastian/code-unit": "^1.0.8",
         "vimeo/psalm": "^4.7"
     },
     "suggest": {
         "ext-pcov": "Install PCov extension to generate code coverage.",
-        "ext-xdebug": "Install Xdebug to generate phpspec code coverage."
+        "ext-xdebug": "Install Xdebug to generate phpspec code coverage.",
+        "sebastian/code-unit": "Install code-unit to support @covers annotations in tests."
```

Comment:
Maybe adding a not in the README about the `@covers` support and how to use it can help users.